tests/test_pipelines.py

```diff
@@ -0,0 +1,227 @@
+from copy import deepcopy
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from zyte_common_items import Article, ArticleListPage, Product, ProductNavigation
+from zyte_common_items.pipelines import DropLowProbabilityItemPipeline
+
+
+@pytest.mark.parametrize(
+    "thresholds_settings, default_threshold, expected_thresholds, expected_default_thresholds",
+    [
+        ({}, 0.09, {}, 0.09),
+        (
+            {
+                "zyte_common_items.Article": 0.2,
+                "zyte_common_items.Product": 0.3,
+            },
+            None,
+            {
+                Article: 0.2,
+                Product: 0.3,
+            },
+            None,
+        ),  # thresholds for items per Item's path
+        (
+            {Article: 0.4, Product: 0.5},
+            None,
+            {
+                Article: 0.4,
+                Product: 0.5,
+            },
+            None,
+        ),  # thresholds for items per Item's class
+        (
+            {Article: 0.4},
+            0.09,
+            {
+                Article: 0.4,
+            },
+            0.09,
+        ),  # default threshold is taken from DEFAULT_ITEM_PROBABILITY_THRESHOLD
+        (
+            {"default": 0.6, Article: 0.4},
+            0.09,
+            {
+                Article: 0.4,
+            },
+            0.6,
+        ),  # default threshold is taken from settings with the key 'default'
+        (
+            {"default": 0, Article: 0.4},
+            0.09,
+            {
+                Article: 0.4,
+            },
+            0,
+        ),  # default threshold is taken from settings with the key 'default' == 0
+    ],
+)
+def test_init_thresholds(
+    thresholds_settings,
+    default_threshold,
+    expected_thresholds,
+    expected_default_thresholds,
+):
+    scrapy = pytest.importorskip("scrapy")  # noqa
```

Comment:
What do you think about moving it to the module level?

Reply:
Yes, sure, let's move!